cluster-configuration/cluster-configuration.yaml

```diff
@@ -50,7 +50,8 @@ machine-sku:
 
```

Comment:
I suggest we use 3 yaml files, each maps to one of the 3 layers of configurations:

- `infrastructure.yaml` --> SKUs and machines.
- `kubernetes.yaml` --> K8s configurations and role definitions, taking `infrastructure.yaml` only as its dependency.
- `pai.yaml` --> Configuration of services, taking `kubernetes.yaml` only as its dependency.

Reply:
Actually, k8s-role-definition should be transparent for user. And user only configure 3 files now.